src/xo-app/vm/tab-stats.js

```diff
@@ -99,7 +97,7 @@ export default injectIntl(
                 <div className='form-group'>
                   <label>{_('useCombinedValuesOnStats')}</label>
                   {' '}
-                  <Toggle value={useCombinedValues} onChange={this._handleCombinedValues} />
+                  <Toggle defaultValue={useCombinedValues} onChange={this.linkState('useCombinedValues')} />
```

Comment:
Keep using controlled components but make sure the value is defined: `!!useCombinedValues`.